src/app/modules/shared/components/month-picker/month-picker.component.spec.ts

```diff
@@ -62,7 +62,7 @@ describe('MonthPickerComponent', () => {
   it('selectMonth should call selectDate', () => {
     spyOn(component, 'selectDate');
     component.selectMonth(10);
-    expect(component.selectDate).toHaveBeenCalledWith(10, 2020);
+    expect(component.selectDate).toHaveBeenCalledWith(10, 2021);
```

Comment:
What would happen next year? would we have to change? Wouldn't it be better to add a moment with a current year?

Reply:
you are right, i have already corrected it